src/metabase/models/permissions.clj

```diff
@@ -1079,6 +1079,33 @@
     (when (and has-native-perms? (not (group-has-native-perms? group-or-id database-or-id)))
       (grant-native-readwrite-permissions! group-or-id database-or-id))))
 
+; Audit Permissions helper fns
+
+(defenterprise default-audit-db-id
+  "OSS implementation of `audit-db/default-audit-db-id`, which is an enterprise feature, so does nothing in the OSS
+  version."
+  metabase-enterprise.audit-db [] ::noop)
+
+(defenterprise default-audit-collection-entity-id
+  "OSS implementation of `audit-db/default-audit-collection-entity-id`, which is an enterprise feature, so does nothing in the OSS
+  version."
+  metabase-enterprise.audit-db [] ::noop)
+
+(defn check-audit-db-permissions
```

Comment:
Should be checking the prefix of each change here?

Reply:
What do you mean? I think this looks good, we just want to make sure the audit DB ID isn't in the incoming graph

Reply:
Don't we also want to make sure that the tables within the app DB permissions aren't changed too. Unsure if my understanding is correct here.